shared-data/opt/local/osbs/osbs.conf

```diff
@@ -26,3 +26,29 @@ token = OSBS_TOKEN
 client_config_secret = client-config-secret
 reactor_config_secret = reactor-config-secret
 can_orchestrate = true
+
+[scratch]
+authoritative_registry = registry.fedoraproject.org
+build_image = vrutkovs/fedora-buildroot:updates-testing
+build_type = prod
+distribution_scope = public
+koji_certs_secret = kojisecret
+koji_hub = https://KOJI_HUB_IP:8083/kojihub
+koji_root = https://KOJI_HUB_IP:8083/kojifiles
+namespace = osbs
+openshift_url = https://OPENSHIFT_IP:8443/
+registry_api_versions = v1,v2
+registry_secret = v2-registry-dockercfg
+registry_uri = https://REGISTRY_IP:5000/v2
+source_registry_uri = http://registry.fedoraproject.org
+sources_command = fedpkg sources
+use_auth = true
+use_kerberos = false
+vendor = Red Hat, Inc.
+verify_ssl = false
+token = OSBS_TOKEN
+client_config_secret = client-config-secret
+reactor_config_secret = reactor-config-secret
+can_orchestrate = true
+scratch = True
+unique_tag_only = True
```

Comment:
`unique_tag_only` is no longer needed. It actually has been removed from osbs-client altogether.